index.html

```diff
@@ -125,10 +125,7 @@ <h1 class="text-white font-rajdhani text-left text-xl md:text-3xl">
                                     iii. Perpetration of fraud against Mr. Piecer's Datalix account.
                                 </li>
                                 <li>
-                                    iv. Baseless accusations directed towards HolaClient and its personnel.

```

Comment:
Nuh uh, You did it

Reply:
Well, not against HC only Demon and I regret it.